querybook/webapp/components/QueryEditor/QueryEditor.tsx

```diff
@@ -480,7 +472,10 @@ export const QueryEditor: React.FC<
 
         const onTextHover = useCallback(
             async (editor: CodeMirror.Editor, node, e, pos, token) => {
+                // Debounce asynchronous checks with a longer delay (e.g. for requesting table metadata)
                 onTextHoverLongDebounce(editor, node, e, pos, token);
+
+                // Faster checks use a shorter delay
```

Comment:
why the suggestion requires a faster check?

Reply:
The UI looks pretty laggy with the 600ms check (and since the function to get the suggestion isn't async, I thought a faster check would be fine)